consensus/snapshot/snapshot.c

```diff
@@ -91,14 +92,19 @@ retcode_t iota_snapshot_init(snapshot_t *const snapshot,
   snapshot->index = 0;
   snapshot->state = NULL;
 
-  // TODO signature validation
-  // String snapshotFile = config.getSnapshotFile();
-  // if (!config.isTestnet() &&
-  //     !SignedFiles.isFileSignatureValid(
-  //         snapshotFile, config.getSnapshotSignatureFile(), SNAPSHOT_PUBKEY,
-  //         SNAPSHOT_PUBKEY_DEPTH, SNAPSHOT_INDEX)) {
-  //   throw new RuntimeException("Snapshot signature failed.");
-  // }
+  if (!testnet) {
```

Comment:
testnet is also signed, we might want to have a separate flag for skipping the sig check?

Reply:
I thought it was signed too, but IRI team told me it's not.
They pushed a testnet sigfile for consistency but it's not the actual signature.

Reply:
:+1: